python/ngraph/ops.py

```diff
@@ -202,6 +202,18 @@ def reshape(node, output_shape, input_order=None, name=None):
     return Reshape(node, AxisVector(input_order), Shape(output_shape))
 
 
+@nameable_op
```

Comment:
Let's move this function to another place in the file, outside of the section marked as `# Unary ops`